tpm2/pcr_profile.go

```diff
@@ -77,34 +79,100 @@ func (l pcrValuesList) copy() (out pcrValuesList) {
 	return
 }
 
+// pcrProtectionProfileInstr is a building block of PCRProtectionProfile.
+type pcrProtectionProfileInstr interface {
+	run(context *pcrProtectionProfileExecContext) error
+}
+
+type pcrProtectionProfileInstrList []pcrProtectionProfileInstr
+
```

Comment:
I think the definitions of type pcrProtectionProfileInstrHandler and pcrProtectionProfileExecContext should come here for clarity